learn/releasenotes/7-18-0.adoc

```diff
@@ -0,0 +1,47 @@
+== Enhanced process instance diagram view
+
+In this release, the process diagram view has been enhanced to show extra information regarding the number of node
+instances (either completed or in progress) for a specific node in the process. This new feature is displayed in form of
+counter badges below each node in the process diagram view. It is also possible to toggle the badges visibility to show
+ or hide it.
+
+== Upgrade of SpringBoot to 2.1.1
+
+jBPM 7.18 comes with long awaited SpringBoot upgrade. Business applications are now based on 2.1.1.RELEASE
+of SpringBoot. All components shipped with jBPM have been tested and confirmed to work without issues after
+upgrade.
+
+NOTE: When upgrading your business application to 7.18, make sure to increase the version of spring boot as well.
+
+[source,xml]
+----
+<parent>
+  <groupId>org.springframework.boot</groupId>
+  <artifactId>spring-boot-starter-parent</artifactId>
+  <version>2.1.1.RELEASE</version>
+</parent>
+----
+
+== Upgrade of Hibernate to 5.3.6
+
+Following up on the recent upgrade of the run time (Wildfly 14), with jBPM 7.18, Hibernate has been upgraded to match
+this run time. Since Hibernate 5.3.x is already on JPA 2.2, it resulted in a split in the KIE Server `WAR` file:
+
+* ee8 - Comes with upgraded Hibernate and targets WildFly 14+ and EAP 7.2
+* ee7 - Comes with Hibernate version as in 7.17 (5.1.x) and targets environments like WAS and WLS
+* webc - Comes with Hibernate 5.3.6 and targets Tomcat
+
+Please ensure you run the right version of the `WAR` file for your run time.
+
+== New development mode for KIE Server
+
+Added ability to KIE Server to start in development mode to provide a more fluent experience during the development
+and testing phases of projects. Development mode provides a more flexible deployment policy that enables you to update
+existing containers while keeping the active process instances for small changes, or to reset the container state
```

Comment:
What does `reset` mean in this context?

Reply:
I believe that means redeploy with abort of process instances...